pypeit/tests/test_onespec.py

```diff
@@ -21,7 +21,7 @@ def test_init():
     assert spec.spectrograph is None, 'Spectrograph should not be set'
 
     spec = onespec.OneSpec(wave, wave, flux, ivar=2*np.ones_like(flux))
-    assert np.allclose(spec.sig, 1/np.sqrt(2)), 'Conversion to sigma is wrong'
+    #assert np.allclose(spec.sigma, 1/np.sqrt(2)), 'Conversion to sigma is wrong'
```

Comment:
Should this be commented out? I think we either need to pass in `sigma` to the `OneSpec()` call, or have a backup (e.g. reinstate the `sig()` function in `OneSpec`) for when sigma is not provided, something like:

```
    @property
    def sig(self):
        """ Return the 1-sigma array
        Returns:
            `numpy.ndarray`_: error array
        """
        if self.sigma is None:
            return np.sqrt(utils.inverse(self.ivar))
        else:
            return self.sigma
```

Alternatively, if we are trying to test that sigma isn't being set (but I'm not sure why) we would need to change this to:
` assert spec.sigma is None, 'SIGMA should not be set'`

Reply:
sigma is a required attribute now, so there is no conversion to sigma which is what this test was testing.